FHIR/GoFSH
# Cimpl 919 support passing fhir version as arg

Fixes #177 

To test:
Try out the -u option, which lets the user decide what FHIR version to use.
For example: `ts-node src/app.ts [path_to_file] -u 5.0.0`

Suggestions for testing: try an empty input file, an populated file with missing FHIR Version, and a file where the file FHIR version is different from the input FHIR version. 

Warning: this comes pre-merged with Julia's changes, so take a look at `app.ts`, `FHIRProcessor.ts`, and `ConfigurationEtractor.ts` if you want the main changes regarding the `-u` option. 

One more note: should the error on line 48 in `ConfigurationEtractor.ts` be a warning or is it good as an error? Would appreciate your input. (I'm personally leaning towards warn since GoFSH can still work with a differing input FHIR version in some cases.)
